lightkurve/lightcurvefile.py

```diff
@@ -159,6 +171,9 @@ def __init__(self, path, quality_bitmask='default', **kwargs):
         elif filetype is None:
             warnings.warn("Given fits file not recognized as Kepler or TESS "
                           "observation.", LightkurveWarning)
+        elif filetype == 'KeplerLightCurveFile-interact':
+            warnings.warn("Custom-made Kepler fits files have limited "\
+                          "functionality.", LightkurveWarning)
```

Comment:
I'm not sure this warning is particularly useful (because it is not actionable).  Is there a specific feature that will not work?  If so, I think we should only have a warning when a non-functioning feature is called?